mpmath/libmp/libmpf.py

```diff
@@ -1157,9 +1161,23 @@ def round_digits(digits, dps, base, rounding=round_nearest):
     return digits, exponent
 
 
+def abs_rounding(rounding, sign):
```

Comment:
maybe this belongs to the round_digits?

Reply:
Maybe, this would add a parameter to round_digits, but could make the code clearer.